MaterialDesignThemes.Wpf/Flipper.cs

```diff
@@ -143,18 +143,6 @@ public bool IsFlipped
             instance.RaiseEvent(args);
         }
 
-        public static readonly DependencyProperty UniformCornerRadiusProperty = DependencyProperty.Register(
-            nameof(UniformCornerRadius), typeof(double), typeof(Flipper), new PropertyMetadata(default(double)));
-
-        /// <summary>
-        /// Gets or sets the (uniform) corner radius applied the the <see cref="Flipper"/> when the MaterialDesignCardFlipper style is applied.
-        /// </summary>
-        public double UniformCornerRadius
```

Comment:
Rather than delete this, lets mark it with `[Obsolete]` and I can add it to the [list of breaking changes for the 5.x release](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/2255).

Reply:
O opps, this was my mistake.